src/chapter3/01-a-http-primer.md

```diff
@@ -0,0 +1,226 @@
+## A HTTP primer
+
+### HTTP and Express.js
+
+When writing a web application, the most common way to get data from a server to a client is via the HTTP procotol (usually augmented by SSL resulting in the HTTPS procotol).
+HTTP is a **request/response protocol**, i.e. HTTP clients send requests to an HTTP server and receive a response in return.
+You have in fact already used the HTTP protocol.
+Every time you browse the internet, HTTP requests are sent under the hood.
+
+We could use the built-in HTTP module of Node.js, but this is not terribly convenient.
+Instead we will use an extremely popular framework called **Express.js**.
```

Comment:
its just called Express

Reply:
its called both, but the official name is Express, yes